larray_editor/utils.py

```diff
@@ -390,3 +405,94 @@ def __getitem__(self, key):
 
     def __len__(self):
         return self.length
+
+
+def replace_inf(value):
+    """Replace -inf/+inf in array with respectively min(array_without_inf)/max(array_without_inf).
```

Comment:
I guess this is a function dedicated to be used internally, users should not know about it. Right?

Reply:
indeed, why do you ask?

Reply:
It is fully documented

Reply:
it does not hurt to document internal functions, right? :)

Reply:
right